content/docs/user-guide/build/distributable-engine.md

```diff
@@ -224,3 +224,21 @@ Engine developers will work on O3DE source code and create new distributable bui
 Project developers will work on project-specific source code or use O3DE creative tools to add project content. To enable their workflows, project developers are required to _only_ pull a project from its hosted location.
 
 Whenever working on their project, project developers should launch the tools that were checked in as the distibutable engine build. These tools are located in the project's `install\bin\Windows\profile\Default` subdirectory.
+
+### License file generation
+
+License attribution files can be generated during the project development process to properly attribute any code or packages that were imported (often called the NOTICES file). To scan project directories for licenses, project developers can run a script located in the engine's `scripts\license_scanner` subdirectory.
+
+In this example, with the engine, project, and `C:\o3de-source\3rdParty` as your 3rd Party packages folder, run the following script in the engine source folder:
```

Comment:
For consistency with other parts of the website, let's use a different name for the packages folder.
```suggestion
In this example, with the engine, project, and `C:\o3de-source\3rdParty` as your downloadable packages folder, run the following script in the engine source folder:
```